core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -619,6 +663,44 @@ object Chunk
     override def map[O2](f: O => O2): Chunk[O2] = indexedSeq(s.map(f))
   }
 
+  /** Creates a chunk from a mutable `java.util.List`. */
+  def javaList[O](javaList: ju.List[O]): Chunk[O] =
+    javaList match {
+      case chunkAsJavaList: ChunkAsJavaList[O] =>
+        chunkAsJavaList.chunk
+
+      case randomAccess: ju.RandomAccess =>
+        new JavaListChunk(randomAccess)
```

Comment:
Just to double check: `java.util.List` is a mutable thing by design. Is it ok to wrap it without making a copy?

Reply:
@satorg I had the same doubt and spent a while looking if there was any way to know if a `ju.List` was **unmodifiable**, even using _reflection_. But no, **Java** things that is an implementation detail nobody would ever care for e.e 😡 

Anyways, ranting aside, it seems most `Chunk` factories wrap mutable things without copying. So I guess it is fine per the philosophy of the library.
I assume the rationale is mostly performance and let users take care of it.

Reply:
Yeah, we intentionally don't do defensive copies for performance reasons. See [this talk](https://mpilquist.github.io/blog/2021/11/26/fs2/) for some details.